CMakeLists.txt

```diff
@@ -108,9 +108,9 @@ if (NOT CMAKE_BUILD_TYPE)
   set ( CMAKE_BUILD_TYPE Release )
 endif()
 
-set ( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -fno-rtti -fvisibility-inlines-hidden" )
```

Comment:
we should set a CXX standard elsewhere, possibly with modern cmake commands

Reply:
Do you mean something like [this](https://cmake.org/cmake/help/latest/prop_tgt/CXX_STANDARD.html)?

Reply:
More like [`CMAKE_CXX_STANDARD`](https://cmake.org/cmake/help/latest/variable/CMAKE_CXX_STANDARD.html#variable:CMAKE_CXX_STANDARD)

Reply:
Should be solved